django_filters/rest_framework/backends.py

```diff
@@ -62,3 +62,10 @@ def to_html(self, request, queryset, view):
         }
         template = loader.get_template(self.template)
         return compat.template_render(template, context)
+
+    def get_fields(self, view):
```

Comment:
Indeed, this is used by the schema generation to introspect what fields are required by a filter class. It's not formalized in the docs yet, so we should discuss prior to any 1.0 django-filter / 3.5 rest framework releases.